cms/static/js/views/list_item.js

```diff
@@ -22,6 +22,7 @@ define([
         canDelete: false,
 
         initialize: function() {
+            this.restrictEditing = this.options.restrictEditing || false;
```

Comment:
Can you write up a description of how/when default values are provided for restrictEditing? For example, here a default value of false is supplied, but in list.js there is not a default value (and the documentation doesn't say that it is optional).